Sources/Endpoints/Communicator/CommunicatorResponse.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CommunicatorResponse.swift
+//  Endpoints
+//
+//  Created by Simon Jarbrant on 2019-06-13.
+//
+
+import Foundation
+
+public struct CommunicatorResponse<Body> {
+    public let headers: [AnyHashable: Any]
+    public let code: Int
+    public let body: Body
+
+    public init(headers: [AnyHashable: Any] = [:], code: Int, body: Body) {
```

Comment:
Är det inte praxis att ha default parametrar sist?

Reply:
I agree that it's more common to have them last. I'll change that in a new commit. 

As a side note, please try to use English when interacting on our open source projects as it makes it easier for non-Swedish speakers to understand.

Reply:
Agreed, I’ll keep that in mind.